src/System Application/App/Edit in Excel/src/Filters/EditInExcelFilters.Codeunit.al

```diff
@@ -19,27 +19,58 @@ codeunit 1490 "Edit in Excel Filters"
     var
         EditinExcelFiltersImpl: Codeunit "Edit in Excel Filters Impl.";
 
+
+#if not CLEAN25
     /// <summary>
     /// Add the specified field using 'and' collection type.
     /// </summary>
     /// <param name="ODataFieldName">The OData name of the field referenced.</param>
     /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+#pragma warning disable AL0432
     procedure AddField(ODataFieldName: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type"): Interface "Edit in Excel Field Filter"
+#pragma warning restore AL0432
     begin
         exit(EditinExcelFiltersImpl.AddField(ODataFieldName, "Edit in Excel Filter Collection Type"::"and", EditInExcelEdmType));
     end;
+#endif
 
+    /// <summary>
+    /// Add the specified field using 'and' collection type.
+    /// </summary>
+    /// <param name="ODataFieldName">The OData name of the field referenced.</param>
+    /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+    procedure AddFieldV2(ODataFieldName: Text; EditInExcelEdmType: Enum "Edit in Excel Edm Type"): Interface "Edit in Excel Field Filter v2"
+    begin
+        exit(EditinExcelFiltersImpl.AddField(ODataFieldName, "Edit in Excel Filter Collection Type"::"and", EditInExcelEdmType));
+    end;
+
+#if not CLEAN25
     /// <summary>
     /// Add the specified field with a specified collection type.
     /// </summary>
     /// <param name="ODataFieldName">The OData name of the field referenced.</param>
     /// <param name="EditInExcelFilterOperatorType">Specifies whether filters for this field have 'and' or 'or', such as Field1 = a|b|c (or operator) or Field1 = &lt;a&amp;>b&gt;amp;>c (and operator). Both operators for the same field is currently not supported.</param>
     /// <param name="EditInExcelEdmType">The Edm type of the OData field as specified in the $metadata document.</param>
+#pragma warning disable AL0432
     procedure AddField(ODataFieldName: Text; EditInExcelFilterOperatorType: Enum "Edit in Excel Filter Collection Type"; EditInExcelEdmType: Enum "Edit in Excel Edm Type"): Interface "Edit in Excel Field Filter"
```

Comment:
Add obsoletion tag. Helps to guarantee you have all `AddField` references removed, even though this is an internal method.